R/osmapiR_connection.R

```diff
@@ -158,9 +158,18 @@ authenticate_osmapi <- function() {
 #' @rdname authenticate_osmapiR
 #' @export
 logout_osmapi <- function() {
-  req <- httr2::request(base_url = get_osmapi_url())
-  req <- oauth_request(req)
-  req$policies$auth_oauth$cache$clear()
+
+  if (utils::packageVersion("httr2") >= "1.0.5.9000") {
+    getFromNamespace("oauth_cache_clear", "httr2")(
```

Comment:
I can't find the function `oauth_cache_clear()` in the httr2 repo, @hadley. Where is the commit/code?

Reply:
Oops, forgot to push it.